.github/workflows/snap.yaml

```diff
@@ -0,0 +1,14 @@
+name: Snap
+
+on:
+  push: {}
```

Comment:
Let's restrict this to certain branches, at least main for now

Reply:
Fixed